app/services/rate_limiter.rb

```diff
@@ -69,6 +69,10 @@ def maxed?
     attempts && attempts >= RateLimiter.max_attempts(rate_limit_type)
   end
 
+  def exceed_max?
+    !expired? && attempts && attempts > RateLimiter.max_attempts(rate_limit_type)
```

Comment:
I looked and looks like the `attempts` will never exceeds the `max`. Check the `increment!` method, it does nothing if `limited?`.

Reply:
I noticed that too, and was part of what I was running into issues with testing. But changing this does fix the test as captured in `spec/controllers/idv/capture_doc_status_controller_spec.rb` in this PR. Hmm.

Reply:
Resolved (or will be) in [this other thread of comments](https://github.com/18F/identity-idp/pull/10538#discussion_r1586655556).